server/auth.go

```diff
@@ -78,15 +91,74 @@ func (h *Handler) LoginPost(c echo.Context) error {
 }
 
 func (h *Handler) LoginGet(c echo.Context) error {
+	sess, _ := session.Get(CookieKeySession, c)
+
```

Comment:
Let's also add a check for `sess.Values["password-expired"]` here int he GET request and return 401

Reply:
I think you meant `ChangeGet`, not `LoginGet`, as this would lead to always `401` on `/auth/login`. Implemented for `ChangeGet`

Reply:
Yes! Sorry about that. Meant `ChangeGet`.